binary-com/deriv-com
# Mohsen / fix dev ssr issue with new gatsby version

Changes:
- with gatsby v4.18.0 DEV_SSR is enabled by default, it causes our
develop environment to break, we have to disable it.

## How To Test it: 
1. please remove the package-lock.json file 
2. install packages with `npm i` command.
3. run `npm run develop` 
4. check if the project runs correctly.

## Note:
the PR contains the updated package-lock.json file with node v14=7.x and npm v8.x


## Type of change

-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Style only
